aws/resource_aws_rds_cluster_instance_test.go

```diff
@@ -52,6 +52,7 @@ func TestAccAWSRDSClusterInstance_basic(t *testing.T) {
 					testAccCheckAWSDBClusterInstanceAttributes(&v),
 					resource.TestMatchResourceAttr("aws_rds_cluster_instance.cluster_instances", "arn", regexp.MustCompile(`^arn:[^:]+:rds:[^:]+:[^:]+:db:.+`)),
 					resource.TestCheckResourceAttr("aws_rds_cluster_instance.cluster_instances", "auto_minor_version_upgrade", "true"),
+					resource.TestCheckResourceAttr("aws_rds_cluster_instance", "copy_tags_to_snapshot", "false"),
```

Comment:
This is missing the resource name and causes the test to fail:

```
--- FAIL: TestAccAWSRDSClusterInstance_basic (702.58s)
    testing.go:538: Step 0 error: Check failed: Check 5/11 error: Not found: aws_rds_cluster_instance in [root]
```

```suggestion
					resource.TestCheckResourceAttr("aws_rds_cluster_instance.cluster_instances", "copy_tags_to_snapshot", "false"),
```

Reply:
thanks.
Fixed it.